### ISSUE TITLE
Parsing fails when there is a notes section (global checksum does not match)

### ISSUE THREAD

**Issue by brian-pantano** [alexdej/puzpy#21]
Some examples are:

https://www.nytimes.com/crosswords/game/daily/2019/03/31
https://www.nytimes.com/crosswords/game/daily/2019/04/02
https://www.nytimes.com/crosswords/game/daily/2019/04/09

I attached .puz and .pdf files. The puz files are gziped since github won't let me attach .puz files directly.

[Mar3119.puz.gz](https://github.com/alexdej/puzpy/files/3255576/Mar3119.puz.gz)
[Apr0919.puz.gz](https://github.com/alexdej/puzpy/files/3255577/Apr0919.puz.gz)
[Apr0219.puz.gz](https://github.com/alexdej/puzpy/files/3255578/Apr0219.puz.gz)
[Mar3119.pdf](https://github.com/alexdej/puzpy/files/3255571/Mar3119.pdf)
[Apr0219.pdf](https://github.com/alexdej/puzpy/files/3255572/Apr0219.pdf)
[Apr0919.pdf](https://github.com/alexdej/puzpy/files/3255573/Apr0919.pdf)

**Comment by brian-pantano** [comment]
Taking a look at the source, it looks like it's because

```
        if self.version.decode(ENCODING) == '1.3' and self.notes:
            cksum = data_cksum(self.notes.encode(ENCODING) + b'\0', cksum)
```

and for these files `self.version.decode(ENCODING)` is 1.4.

`>= '1.3'` will fix it until version 1.10, or you could split on dot, convert to int, and compare.

**Comment by alexdej** [comment]
Thanks for the bug report! I just pushed a fix for it. Will release to pypi soon
